kas/config/common.yaml

```diff
@@ -15,6 +15,10 @@ local_conf_header:
     INIT_MANAGER="systemd"
     TEDGE_CONFIG_DIR = "/etc/tedge"
 
+    # customize tedge features
+    DISTRO_FEATURES:append = " tedge"
```

Comment:
Do we need this anymore? If so, I'm wondering how we can handle the upgrade path for existing users, as I don't think we'll be able to add this to the layer.conf (as that would be bad practice afaik)